.eslintrc.js

```diff
@@ -7,27 +7,22 @@ module.exports = {
     ecmaVersion: 2018,
     sourceType: 'module',
     ecmaFeatures: {
-      legacyDecorators: true
-    }
+      legacyDecorators: true,
```

Comment:
Was anything changed in this file? Seems like only adding commas. If so, we should remove these changes.

Reply:
Nothing was changed in this file. It's probably because my prettier settings in `vs code` is different to yours.

Reply:
We need to add a `trailingComma: 'none',` to `.prettierrc,js` in this repo. I can do that after we get this PR in.

Reply:
Alright then, I'll resolve this conversation.